python/social_bot/models/pr2_noplugin/model.sdf

```diff
@@ -670,6 +671,40 @@
           </clip>
         </camera>
       </sensor>
+      <sensor name='wide_stereo_gazebo_l_stereo_camera_sensor' type='camera'>
+          <always_on>1</always_on>
+          <update_rate>25</update_rate>
+          <pose frame=''>0.0682 0.03 0.1146 0 -0 0</pose>
+          <camera name='__default__'>
+            <horizontal_fov>1.5708</horizontal_fov>
+            <image>
+              <width>640</width>
```

Comment:
Maybe we can use the same dim as _resized_image_size or 2*_resized_image_size here. That will save quite a lot rendering effort, and it would be better to keep the ratio of the images when resizing.

Reply:
good idea.

there are still a few difficulties with the current camera setup
1. object could fall outside the camera view (when PR2 base move, or object move, etc)
2. two stereo camera still hard to detect depth (distance from finger tip to object seems an important feature for previous internal states based training).

Reply:
我在navigation task 用了 <horizontal_fov>2.0944</horizontal_fov>, 120°
实在不行可以考虑fish-eye like fov, 160°-175°
深度的话简单的把两幅图拼起来然后cnn可能不太利于提取深度信息, 输入部分的网络结构可以考虑借鉴Flow-Net S or PWC-Net的network前面的部分.